remediation/code-review/02-countValue.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Given an array and a value, return the number of times that value is found
+ * in the array.
+ *
+ * @example
+ * countValue([1, 2, 3, 4, 5], 2); // => 1
+ * countValue([1, 2, 3, 4, 5], 17); // => 0
+ * countValue([1, 2, 1, 2, 3, 4, 1, 2, 1], 1); // => 4
+ * countValue([10, 10, 10, -10], 10); // => 3
+ * countValue(['hello', bananas', 'hello'], 'hello'); // => 2
+ * countValue(['hello', bananas', 'hello'], 'giraffe'); // => 0
+ *
+ * @param {object[]} array - An array
+ * @param {object} - A value to count
+ * @returns {number} The number of times the value appears in the array
+ */
+function countValue(array, value) {
+  for(i=0;i<array.length;i++)
+  {
+    if(array[i] != value)
+    {
+      array.splice(i,1)
```

Comment:
This will result in the same issue as your `countValue` answer. The good news is that means you probably can apply the same fix to both!